csrc/python_frontend/python_bindings.cpp

```diff
@@ -2282,7 +2282,7 @@ void initNvFuserPythonBindings(PyObject* module) {
       },
       py::arg("arg"),
       py::arg("axes"),
-      py::arg("correction"),
+      py::arg("correction") = 1,
```

Comment:
Added default parameter to match pytorch signature.
https://pytorch.org/docs/stable/generated/torch.var_mean.html